tools/.golangci.yaml

```diff
@@ -23,7 +23,7 @@ linters:
     - staticcheck
     - stylecheck
     - unused
-    - unconvert  # Remove unnecessary type conversions
+    - unconvert # Remove unnecessary type conversions
```

Comment:
No need to touch this.

Reply:
That's getting changed by `make fix`.

Reply:
Hmm, @fuweid do you know why?
Can you send a separate PR?

Reply:
@dusk125 Thanks! Let me fix this